.github/PR_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,24 @@
+## What changed / motivation ?
+
+Please include relevant motivation and context
+
+## Linked PR/Issues
+
+Fixes # (issue)
+
+## Additional Context
+
+<!--- Screenshots, Tests, Breaking Change, Anything Else ? --->
+
+Screenshots, Tests, Anything Else
+
+## Pre-flight checklist
+
+- [ ] I have read the contributing guidelines
+      [Contribution Guidelines](../CONTRIBUTING.md)
+- [ ] I have signed the contributing agreements
+- [ ] I have written unit tests where necessary (If it is a feature commit)
```

Comment:
Let's remove these two bullet points since they're redundant.
CLA signing is already checked by a signal. And Unit tests are covered in the `Additional Context` section above.